cmd/main.go

```diff
@@ -34,7 +35,7 @@ func main() {
 	accountSvc := account.NewAccountService(conn, clock)
 
 	r := gin.Default()
-
+	r.Use(cors.Default())
```

Comment:
Can we take these values from a `.env` file or something? unless we want CORS to be enabled for everything even on production

Reply:
I think for now we don't have to do that, since we still need to confirm a few things with KG regarding the production environments and configs

This is just a quick setup to prevent further blocking I suppose, wdyt

Reply:
But yeah good call, I'll put that on the backlog regarding what to change for the production setup, thanks